index.html

```diff
@@ -130,14 +168,14 @@ <h3 class="dashbord1">Profesional Art Printing Data</h3>
             </div>
         </div>
 
-        <div class="box3" tabindex="0">
+        <div class="box3 click-card" tabindex="0">
             <h3 class="dashbord">Data Dashboard Healthcare</h3>
             <p class="texto3">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
 
             <div class="botones3">
                 <ul>
                     <li>
-                        <bUtton type="button">HTML</bUtton>
+                        <button type="button">HTML</bUtton>
                     </li>
                     <li> <button type="button">BootStrap</button></li>
                     <li> <button type="button">Ruby</button></li>
```

Comment:
- [ ] Following the project requirements, The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.**
- Check out the requirements below 👇 :
<img width="757" alt="image" src="https://user-images.githubusercontent.com/77190203/162258170-aeaa7064-760a-48cf-a7da-390024fb3308.png">

Reply:
Thank you very much for  the CR